src/schema/v2/CollectorProfile/collectorProfiles.ts

```diff
@@ -0,0 +1,59 @@
+import { GraphQLFieldConfig, GraphQLID, GraphQLString } from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import {
+  connectionWithCursorInfo,
+  paginationResolver,
+} from "../fields/pagination"
+import { CollectorProfileType } from "./collectorProfile"
+
+export const CollectorProfilesConnection: GraphQLFieldConfig<
+  void,
+  ResolverContext
+> = {
+  type: connectionWithCursorInfo({
+    nodeType: CollectorProfileType,
+  }).connectionType,
+  args: pageable({
+    partnerID: {
+      type: GraphQLID,
+    },
+    term: {
+      type: GraphQLString,
+      description: "If present, will search by term",
+    },
+  }),
+  description:
+    "A list of collector profiles that have sent an inquiry to a partner",
+  resolve: async (
+    _root,
+    { term, partnerID, ...args },
+    { collectorProfilesLoader }
+  ) => {
+    if (!collectorProfilesLoader)
+      throw new Error(
+        "A X-Access-Token header is required to perform this action."
+      )
+
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+    const { body, headers } = await collectorProfilesLoader({
```

Comment:
Would it make sense to add a check for the required args (partnerId and term) before this?

Reply:
It does, indeed! 
Addressed and tested on [310dc2f](https://github.com/artsy/metaphysics/pull/5044/commits/310dc2f27dcfe25e2753d6afd6a5ddd0606b380f).